src/index.html

```diff
@@ -1,21 +1,67 @@
-<!doctype html>
-<html lang="en">
-  <head>
-    <meta charset="UTF-8">
-    <meta 
-      name="viewport"
-      content="
-        width=device-width, 
-        user-scalable=no, 
-        initial-scale=1.0, 
-        maximum-scale=1.0, 
-        minimum-scale=1.0
-      "
-    >
-    <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
-  </head>
-  <body>
-    <h1>Product cards</h1>
-  </body>
-</html>
+<!DOCTYPE html>	
+<html lang="en">	
+  <head>	
+    <meta charset="UTF-8">	
+    <meta	
+      name="viewport"	
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"	
+    >	
+    <link	
+      rel="preconnect"	
+      href="https://fonts.googleapis.com"	
+    >	
+    <link	
+      rel="preconnect"	
+      href="https://fonts.gstatic.com"	
+    >	
+    <link	
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"	
+      rel="stylesheet"	
+    >	
+    <title>Product cards</title>	
+    <link rel="stylesheet" href="./style.css">	
+  </head>	
+  <body>	
+    <div class="card" data-qa="card">	
+
+      <img 
+      class="card__image" 
+      src="./images/imac.jpeg" 
+      alt="Apple iMac" 
+      >	
+
+      <h1 class="card__description">	
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)	
+      </h1>	
```

Comment:
Everything went well , but i have few notes :
I think , it is better here to wrap the text with a P tag or a div block instead of h1 tag which is by convention for the main title , and it has some default properties as well.